style.css

```diff
@@ -0,0 +1,3 @@
+.my-color {
+	background-color: cyan;
+	color: blue; }
```

Comment:
[OPTIONAL] It is considered bad practice to use color names, consider changing them to their `hex` or `rgb` equivalents.